eclipse-collections/src/main/java/org/eclipse/collections/impl/set/sorted/mutable/SynchronizedSortedSet.java

```diff
@@ -98,7 +98,7 @@
      * SynchronizedSortedSet that wraps the adapter. Additionally, a developer specifies which lock to use
      * with the collection.
      */
-    public static <E, S extends SortedSet<E>> MutableSortedSet<E> of(S set, Object lock)
+    public static <E, S extends SortedSet<E>> SynchronizedSortedSet<E> of(S set, Object lock)
```

Comment:
Why is the return type changed?

Reply:
@motlin I think I needed it in my first round of changes but don't anymore, I can revert. I left it because it was more precise and not breaking.